contracts/oss-diff.yml

```diff
@@ -4518,11 +4518,11 @@ paths:
                     type: object
                     properties:
                       next:
-                        $ref: '#/components/schemas/ScraperTargetResponse/allOf/1/properties/links/properties/self'
+                        $ref: '#/components/schemas/Task/properties/links/properties/self'
```

Comment:
It seems odd that this has changed, but I expect there is only really one definition of "self" required.